webapp/src/app.tsx

```diff
@@ -1726,6 +1731,9 @@ ${compileService && compileService.githubCorePackage && compileService.gittag ?
         const isExperimentalUrlPath = location.pathname !== "/"
             && (targetTheme.appPathNames || []).indexOf(location.pathname) === -1;
         const showExperimentalBanner = !isLocalServe && isApp && isExperimentalUrlPath;
+        const isWindows10 = pxt.BrowserUtils.isWindows10();
+        const showWindowsStoreBanner = !this.state.hideWindowsStoreBanner && !pxt.winrt.isWinRT() && isWindows10 && pxt.appTarget.appTheme.windowsStoreLink;
+        const liveUrl = pxt.appTarget.appTheme.homeUrl + location.search + location.hash;
```

Comment:
This is a merge conflict. liveUrl is not used here.

Reply:
Fixed